apps/console/src/features/identity-providers/components/wizards/enterprise-idp-create-wizard.tsx

```diff
@@ -814,15 +847,18 @@ export const EnterpriseIDPCreateWizard: FC<EnterpriseIDPCreateWizardProps> = (
                         className="content-container"
                         data-testid={ `${ testId }-modal-content-2` }>
                         { alert && alertComponent }
-                        <Wizard2
-                            ref={ wizardRef }
-                            initialValues={ initialValues }
-                            onSubmit={ handleFormSubmit }
-                            uncontrolledForm={ true }
-                            pageChanged={ (index: number) => setCurrentWizardStep(index) }
-                            data-testid={ testId }>
-                            { resolveWizardPages() }
-                        </Wizard2>
+                        { !isIDPListLoading ?
```

Comment:
Lets use the formatting in [DEVELOPER](https://github.com/wso2/identity-apps/blob/master/docs/DEVELOPER.md#formatting) guide for ternary operators.

Reply:
Addressed this in commit 8e64d82